apps/prairielearn/elements/pl-order-blocks/dag_checker.py

```diff
@@ -39,7 +39,7 @@ def solve_dag(
     contiguously, making it a solution to the given problem
     """
     graph = dag_to_nx(depends_graph, group_belonging)
-    sort = list(nx.topological_sort(graph))
+    sort = list(nx.topological_sort(graph))  # type: ignore
```

Comment:
Short version of why I think these ignores are needed:

NetworkX doesn't have typechecking, so pyright does it's analysis without any verified type hints on the NetworkX functions (everything is inferred). I think it doesn't quite infer the right annotations in some places, and so it complains. I couldn't find a way to have Pyright skip certain modules the way mypy is set.